webpack.config.js

```diff
@@ -31,12 +30,12 @@ const serverConfig = {
   },
 };
 
-const clientConfig = {
+const clientServerDataConfig = {
```

Comment:
Your client configs are extremely similar, consider making a joint shared config object and then extend with the differing objects instead of repeating

Reply:
Changed in latest commit.